bigbluebutton-html5/imports/ui/components/external-video-player/modal/component.jsx

```diff
@@ -133,29 +133,33 @@ class ExternalVideoModal extends Component {
                 name="video-modal-input"
                 value={url}
                 placeholder={intl.formatMessage(intlMessages.urlInput)}
+                disabled={sharing}
               />
             </label>
             <div className={styles.youtubeNote}>
               {intl.formatMessage(intlMessages.note)}
             </div>
           </div>
 
-          <div className={styles.content}>
+          <div>
             {this.renderUrlError()}
           </div>
 
           <Button
-            className={styles.startBtn}
-            label={intl.formatMessage(intlMessages.start)}
-            onClick={this.startWatchingHandler}
-            disabled={startDisabled}
-          />
-
-          <Button
-            className={styles.stopBtn}
-            label={intl.formatMessage(intlMessages.stop)}
-            onClick={this.stopWatchingHandler}
-            disabled={!sharing}
+            className={
+              sharing
```

Comment:
What I had in mind was that you don't even open the modal when you want to Stop sharing YouTube. The action menu triggers the stop command. Otherwise you end up pressing + -> stop -> stop. No need of modal's stop